assets/script.js

```diff
@@ -65,13 +67,28 @@ function getSelectedRadioMood() {
         'input[name="mood"]:checked');
     console.log(selectedMood)
     if (selectedMood) {
-        moodArray.push(selectedMood.value)
+        var selectedRating = moodArray.push(parseInt(selectedMood.value)); // parseInt converts the value to an integer
```

Comment:
push value returns the length of the array and not the value pushed. So we will have to update this later on all 4 surveys. 

Good call on parseInt addition.